maintainers/scripts/README.md

```diff
@@ -0,0 +1,22 @@
+# Maintainer scripts
+
+This folder contains various executable scripts for nixpkgs maintainers,
+and supporting data or nixlang files as needed.
```

Comment:
```suggestion
This folder contains various executable scripts for nixpkgs maintainers,
and supporting data or nixlang files as needed.
These scripts generally aren't a stable interface and may changed or be removed.
```

I think generally files in the source shouldn't be relied upon, so just emphasising this here.